generator/destination.go

```diff
@@ -11,4 +11,7 @@ type Destination interface {
 
 	// Get latest timestamp seen in the destination.
 	GetLatestTimestamp() (time.Time, error)
+
+	// ConfigureDestination is used to make any configuration changes to the destination that might be required for sending documents.
+	ConfigureDestination() error
```

Comment:
do we need this interface? you only use this function after casting to `snowflake`

Reply:
Thanks for pointing this out, earlier we had the ConfigureDestination method only for snowflake but later we decided to add it to the interface to keep things consistent, I have updated main.go to remove the casting.